<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSolicitudes</class>
 <widget class="QWidget" name="WidgetSolicitudes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Usuarios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblUsuarios">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>671</width>
     <height>141</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nombres</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nacimiento</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Solicitudes Recibidas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblSolRec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Solicitudes Enviadas&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblSolEnv">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Solcitudes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
